### ISSUE TITLE
Code the wireframe for USER section

### ISSUE THREAD

**Issue by rajatmw1999** [adavijit/BlogMan#146]
The wireframe for education section is stored at "\front-end\public\wire-frame\User_Profile\1.png".
Be sure to make the design responsive for mobile as well. This is just the header of the education section of blogman.

**Comment by ashishnagpal2498** [comment]
The user's page - blog , favorite , followers and following , should have code consistency. So assigning to different people will get different solutions. Can work on all 4 @rajatmw1999  ?

**Comment by rajatmw1999** [comment]
You are right. I had this thing in mind but giving all 4 issues to one person wont be right. Anyways, you go ahead with this one and i will assign the other 3 to you as well if I think you have done a good job on this one, Until then, the other 3 will remain unassigned.
